src/bastion/examples/hello_world.rs

```diff
@@ -0,0 +1,39 @@
+use bastion::prelude::*;
+
+///
+/// Prologue:
+/// The most classic of all examples and especially the essential hello, world!
+///
+fn main() {
+    // We need bastion to run our program
+    Bastion::init();
+    // We are starting the Bastion program now
+    Bastion::start();
+    // We are creating the group of children which will received the message
+    let children = Bastion::children(|children| {
+        // We are creating the function to exec
+        children.with_exec(|ctx: BastionContext| {
+            async move {
+                msg! {
+                  // We are waiting a msg
+                  ctx.recv().await?,
+                  ref msg: &'static str => {
+                    // We are logging the msg broadcasted bellow
+                    println!("{}", msg);
+                  };
+                  _: _ => ();
+                }
+                // We are stopping bastion here
+                Bastion::stop();
```

Comment:
We might wanna stop the bastion at line 37, before `Bastion::block_until_stopped();`

Otherwise a child will stop the bastion after processing a message, which doesn't seem like something someone would do

Reply:
Yes that's the goal, stop bastion after processing a message. If you try to stop the bastion at line 37, before the `Bastion::block_until_stopped();`, the `msg`is not logged. If you try after, the bastion will never be stopped.